### DISCUSSION TITLE
Any guide of command line options?

### DISCUSSION THREAD

**Discussion by exodehm** [pdfarranger/pdfarranger#771] [General]
Hello:

I have an error reported yet, "don't know how to __ str __ this object". Looks that upgrading pikepdf to 1.11.2 solves it, but I can't to update my PC.
I would like if I call the program from console, I could avoid this error, but I can't see the options to do it.
Thanks!

**Comment by kbengs** [comment]
Hi,

`pdfarranger file1.pdf file2.pdf` will open files.
`pdfarranger -v` will print the version and exit.
The program can take no more options.

What operating system do you use and can you post the version numbers you can see in the about dialog?

**Reply by exodehm** [reply]
Hello.
I have Linux Mint 20.3 Una (5.4.0-135-generic #152-Ubuntu).
My version of PDF Arranger is 1.4.2

Output of _sudo apt list "*pdfarranger*" "*pikepdf*"_:
pdfarranger/focal,focal,now 1.4.2-1 all [instalado, automático]
pikepdf-doc/focal,focal 1.10.3+dfsg-1 all
python3-pikepdf/focal,now 1.10.3+dfsg-1 amd64 [instalado, automático]

Thank you

Edit: I forgot to say that it doesn't happen always. I don't know if the problem is the file or I don't know. I tried to short the filenames and put it in a folder with not a lot of characters (/home/user/pdfarrager/)

Also: I have using PDF Arrange (and PDF Shuffler before) for a years, and today is the first time I get this error.

**Comment by kbengs** [comment]
1.4.2 is old. Many bugs has been fixed since that. Also updates in pikepdf and qpdf has caused issues with older versions of pdfarranger. 
You should be able to install newest pdfarranger with pip if you follow to instructions https://github.com/pdfarranger/pdfarranger#install-from-source

**Reply by exodehm** [reply]
Done. Now all seem works fine. Thank you!
Anyway the pdf it couldn't be merged are done, but it show me a warning message. Maybe it was the cause of the error with the older version. I put because maybe it could be useful:
```
/home/user/.local/lib/python3.8/site-packages/pikepdf/models/metadata.py:387: UserWarning: The metadata field /AAPL:Keywords with value 'pikepdf.Array([  ])' has no XMP equivalent, so it was discarded
  warn(msg)
```
